m3u8/protocol.py

```diff
@@ -25,7 +25,7 @@
 ext_x_cue_out_cont = '#EXT-X-CUE-OUT-CONT'
 ext_x_cue_in = '#EXT-X-CUE-IN'
 ext_x_cue_span = '#EXT-X-CUE-SPAN'
-ext_x_scte35 = '#EXT-OATCLS-SCTE35'
```

Comment:
`EXT-X-SCTE35` is actually a different tag. It's specified in [this document](https://www.scte.org/documents/5003/SCTE_35_2022.pdf).

Reply:
Indeed! Nice catch